TAF/testScenarios/integrationTest/UC_app_service_configurable/external_mqtt_trigger.robot

```diff
@@ -15,29 +15,22 @@ ExternalTrigger001 - Test external mqtt trigger works
     Given Set Test Variable  ${publish_msg}  Message from mqtt publisher
     And Set Test Variable  ${mqtt_trigger_topic}  external-request
     And Set Test Variable  ${mqtt_export_topic}  edgex-export
-    And Subscribe MQTT Broker Topics
-    When Run process  python ${WORK_DIR}/TAF/utils/src/setup/mqtt-publisher.py ${mqtt_trigger_topic} "${publish_msg}"
+    And Subscribe MQTT Broker Topics ${mqtt_trigger_topic} With ${BROKER_PORT}  # Subscribe trigger mqtt broker
+    And Subscribe MQTT Broker Topics ${mqtt_export_topic} With ${EX_BROKER_PORT}  # Subscribe export mqtt broker
+    When Run process  python ${WORK_DIR}/TAF/utils/src/setup/mqtt-publisher.py ${mqtt_trigger_topic} "${publish_msg}" ${BROKER_PORT}
```

Comment:
Publish to external broker for the trigger

```suggestion
    When Run process  python ${WORK_DIR}/TAF/utils/src/setup/mqtt-publisher.py ${mqtt_trigger_topic} "${publish_msg}" ${EX_BROKER_PORT}
```

Reply:
Updated